pac4j-oauth/src/main/java/org/pac4j/oauth/profile/generic/GenericOAuth20Profile.java

```diff
@@ -0,0 +1,89 @@
+package org.pac4j.oauth.profile.generic;
+
+import org.pac4j.core.context.Pac4jConstants;
+import org.pac4j.core.profile.Gender;
+import org.pac4j.oauth.profile.OAuth20Profile;
+
+import java.util.Locale;
+
+/**
+ * <p>This is the user profile for the generic OAuth 2.0 client.</p>
+ * <p>It is returned by the {@link org.pac4j.oauth.client.GenericOAuth20Client}  </p>
+ *
+ * @author aherrick
+ * @since 1.9.2
+ */
+public class GenericOAuth20Profile extends OAuth20Profile {
+
+    private transient GenericAttributesDefinition attributes = new GenericAttributesDefinition();
```

Comment:
I think it's important to remain non-static, so it can be reassigned. In the client, someone can pass in their own attributes definition that extends GenericAttributesDefinition. This allows them to specify that their service returns FAMILY_NAME as something other than "family_name" ("last_name", for example). I can make it final though, if that's what you want